core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -24,6 +24,28 @@ oppia.factory('UrlInterpolationService', ['alertsService',
     return (typeof value === 'string') || (value instanceof String);
   };
 
+  var validateResourcePath = function(resourcePath) {
+    if (!resourcePath) {
+      alertsService.fatalWarning(
+        'Empty path passed in method.');
+      return null;
```

Comment:
Only if appropriate.

Though there's also something to be said for doing just the minimal change, so maybe just do it for any code you've added.